Code/Framework/AzToolsFramework/Tests/Input/QtEventToAzInputMapperTests.cpp

```diff
@@ -512,4 +516,145 @@ namespace UnitTest
             return info.param.m_az.GetName();
         }
     );
+
+    struct MouseMoveParam
+    {
+        AzToolsFramework::CursorInputMode mode;
+        int iterations;
+        QPoint startPos;
+        QPoint deltaPos;
+        QPoint expectedPos;
+        const char* name;
+    };
+
+    class MoveMoveWrapParamQtEventToAzInputMapperFixture
+        : public QtEventToAzInputMapperFixture
+        , public ::testing::WithParamInterface<MouseMoveParam>
+    {
+    };
+
+    TEST_P(MoveMoveWrapParamQtEventToAzInputMapperFixture, MouseMove_NoAzHandlers_VerifyMouseMovmentViewport) 
+    {
+        // setup
+        const MouseMoveParam mouseMoveParam = GetParam();
+
+        AzFramework::InputChannelNotificationBus::Handler::BusConnect();
+        m_captureAzEvents = false;
+        m_inputChannelMapper->SetCursorMode(mouseMoveParam.mode);
+        m_rootWidget->move(100, 100);
+
+
```

Comment:
uber nit: Can we lose the extra new line (nearly always only have max 1, especially inside a function)